Ansible/inventories/inventory.za-chpc

```diff
@@ -14,7 +14,10 @@ grid-ui.chpc.ac.za needs_certificate='false' emi_service='UI' yaim_options='-n U
 
 # Site BDII
 [site-bdiis]
-grid-bdii.c4.csir.co.za needs_certificate='false' emi_service='BDII_site' yaim_options='-n BDII_site'
+grid-bdii.chpc.co.za needs_certificate='false' emi_service='BDII_site' yaim_options='-n BDII_site'
+
+[top-bdiis]
```

Comment:
You should leave out the [top-bdiis] group. You don't have top-bdiis at the site and this is the **site** inventory